src/database.ts

```diff
@@ -0,0 +1,122 @@
+import fs from 'node:fs';
+import dotenv from 'dotenv';
+import { Client, QueryResult } from 'pg';
+import { UserRow, WordRow } from './interfaces/Database';
+import { log } from './utils/log';
+dotenv.config();
+
+const client = new Client({
+	connectionString: process.env.DATABASE_URL,
+	ssl: { rejectUnauthorized: false },
+});
+
+/**
+ * Connect to the database and create the tables if they don't exist
+ */
+export async function setUp(): Promise<void> {
+	client.connect();
+
+	const tableUsers = 'CREATE TABLE IF NOT EXISTS "users" ("id" TEXT, "status" BOOLEAN);';
+	const tableWords = 'CREATE TABLE IF NOT EXISTS "words" ("word" TEXT, "status" BOOLEAN);';
+
+	await client.query(tableUsers)
+		.then(() => log('Users table created/verified', 'DB', 'green'));
+	await client.query(tableWords)
+		.then(() => log('Words table created/verified', 'DB', 'green'));
+}
+
+/**
+ * Check if the user exists in the database. If not, add them. And if they do, return a boolean
+ * indicating if they can play or not
```

Comment:
This looks like a lot of responsability for this function. Ignore this if we're not following the Single-responsability principle.

Reply:
True, I can try to refactor it (it's cofusing anyways).